.github/workflows/sandbox-creating.yml

```
@@ -136,17 +138,34 @@
           aws-region: ${{ env.AWS_REGION }}
 
-      - name: Validate AWS Region
+      - name: Retrieve GitHub Token from Secrets Manager
         run: |
-          if [ -z "${{ vars.AWS_REGION }}" ]; then
-            echo "::error::AWS_REGION is not set"
+          SECRET_ID=$(aws secretsmanager list-secrets \
+            --region "${AWS_REGION}" \
+            --query "SecretList[?starts_with(Name, 'github-token-') && DeletionDate==null].Name | [0]" \
+            --output text)
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)